src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -37,105 +40,89 @@ const defaultProps = {
     },
 };
 
-class CloseAccountPage extends Component {
-    constructor(props) {
-        super(props);
+function CloseAccountPage (props) {
+    const [isConfirmModalVisible, setConfirmModalVisibility] = useState(false);
+    const [reasonForLeaving, setReasonForLeaving] = useState('');
 
-        this.onConfirm = this.onConfirm.bind(this);
-        this.validate = this.validate.bind(this);
-        this.hideConfirmModal = this.hideConfirmModal.bind(this);
-        this.showConfirmModal = this.showConfirmModal.bind(this);
-        CloseAccount.clearError();
-        this.state = {
-            isConfirmModalVisible: false,
-            reasonForLeaving: '',
-        };
-    }
+    useEffect(() => () => CloseAccount.clearError());
```

Comment:
Hmm the linter failed and wanted me to fix this but I think it's kind of confusing looking now, ultimately we want to remove it anyways so I'll leave it up to the reviewers on whether or not we should leave it as is.

Reply:
you can see what it was like before here https://github.com/Expensify/App/pull/19328/commits/3830a3ed00437e48d8b90aced4d9261200e55e10